app/services/userService.js

```diff
@@ -54,7 +54,7 @@ class UserService {
       }
     });
 
-    return fetch(request).then(response => response.json());
+    return fetch(request);
```

Comment:
With this change, this service will return a stream and not a json object. Is Doesn't that break anything?